### PR TITLE
Dev cone proxy

### PR DESCRIPTION
**Pull request by v13abh**
- Added task for enabling Proxy using Apache2 on port 443. 
- Added variable for admin credentials to be the same for both proxy and repox login. 
- Added sub tags to role for easier troubleshooting and redeployment.
- Made repox git clone idempotent.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -20,8 +20,8 @@ repox_path_log: /var/log
 repox_path_tomcat: /usr/share/tomcat
 
 # the user that logs in to the web interface
```
Comment (jsnshrmn) [inline defaults/main.yml:33]: Where are admin_username and admin_password coming from? If your goal is to not have a default (which is perfectly reasonable), then let's not set the vars in this file at all.

**Review by jsnshrmn — COMMENTED** [review]
Gave it a look over, but didn't do any testing.

File: defaults/main.yml (hunk 1 above)
Comment (v13abh) [reply defaults/main.yml:33]: These variables are set in the my-vars.yml file and are used for both the Repox and the Apache Proxy credentials. I made reference to this in the README of the vagrant_repox repo but neglected to make changes to the README in the Role itself...

If we do not set the vars in the defaults/mail.yml file of the Role, where would be the best place to set them?

Comment (jsnshrmn) [reply defaults/main.yml:33]: Did some var cleanup per our discussion.

**Pull request merged**